src/mafiasi_link_shortener/frontend/mafiasi_link_shortener/src/api/apis/LinksApi.ts

```diff
@@ -62,6 +62,14 @@ export interface LinksUpdateRequest {
  *
  */
 export class LinksApi extends runtime.BaseAPI {
+    get_headers() {
+        let headers: runtime.HTTPHeaders = {};
+        const csrf_token = document.cookie.split('; ').find(row => row.startsWith('csrftoken'))?.split('=')[1]
+        if (csrf_token) {
+            headers['X-CSRFToken'] = csrf_token;
+        }
+        return headers;
+    }
```

Comment:
Please don't modify the code that is generated automatically from the API schema.

You can regenerate it with the [`gen-api-client`](https://github.com/fsinfuhh/mafiasi_link_shortener/blob/1eb17a9484165d4aab624cbfa958790abb3b175d/src/mafiasi_link_shortener/frontend/mafiasi_link_shortener/package.json#L13) npm script

Reply:
But then how can I set the csrf header?

Reply:
Done in 7611768a15a98b74c641d6f5d16ddfccaff72112.